ts/components/MemberListItem.tsx

```diff
@@ -102,8 +102,8 @@ const StyledCheckContainer = styled.div`
   align-items: center;
 `;
 
-type MemberListItemProps = {
-  pubkey: string;
+type MemberListItemProps<T extends string> = {
```

Comment:
Why the generic?

Reply:
so we have typed string, depending on the usecase. for instance, passing a PubkeyType would make the onClick callbacks also have a PubkeyType, instead of just string as it used to be.

Reply:
Ah okay. Makes sense. Resolved